runtime/lua/vim/fs.lua

```diff
@@ -266,6 +266,7 @@ function M.find(names, opts)
   vim.validate('type', opts.type, 'string', true)
   vim.validate('limit', opts.limit, 'number', true)
   vim.validate('follow', opts.follow, 'boolean', true)
+  vim.validate('dfs', opts.dfs, 'boolean', false)
```

Comment:
Placeholder name. Python uses the term "topdown" for this option but I kinda don't get it, so don't like it. Might go with "depth", "depth_first" or "order".